airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1198,6 +1198,14 @@ def __init__(
         impersonation_chain: str | Sequence[str] | None = None,
         **kwargs,
     ) -> None:
+        if bigquery_conn_id:
```

Comment:
It seems `bigquery_conn_id` might not exist. Should we try something like `kwargs.get("bigquery_conn_id", None)`?